src/js/module.js

```diff
@@ -262,6 +269,35 @@ iotjs_module_t.load = function(id, parent) {
 };
 
 
+function _makeRequireFunction(mod) {
+  var Module = mod.constructor;
+  function require(id) {
+    return mod.require(id);
+  }
+
+  function _resolve(request) {
+    if (!util.isString(request)) {
```

Comment:
Is there any reason preferring `util.isString` instead of  `typeof request === 'string'`?

Reply:
it is same effect but I will modify.